bindings/python/iota_sdk/types/transaction_options.py

```diff
@@ -39,10 +47,18 @@ class RemainderValueStrategy(Enum):
         ChangeAddress: Allows to move the remainder value to a change address.
         ReuseAddress: Allows to keep the remainder value on the source address.
     """
-    ChangeAddress = None,
-    ReuseAddress = None,
+    ChangeAddress = None
+    ReuseAddress = None
 
     def as_dict(self):
+        """
+        The function `to_dict` returns a dictionary with the strategy name and its corresponding value.
```

Comment:
And also remove the actual function name, there is no point putting it and this one is a good example that it is error-prone